phpstan.neon

```diff
@@ -0,0 +1,11 @@
+parameters:
+    level: max
+    checkMissingIterableValueType: false
+    paths:
+        - src
+        - tests
+    ignoreErrors:
+        - '#Parameter \#2 \.\.\.\$values of function sprintf expects bool\|float\|int\|string\|null, mixed given\.#'
```

Comment:
Kde to rve? Nebylo by lepsi to osetrit v kodu?

Reply:
https://github.com/keboola/azure-key-vault-php-client/blob/0be1812d7c7c33c04678479c1f640271dddd60dd/src/GuzzleClientFactory.php#L69 nevím no moc jak to ošetřit a úplně se na tom nezajebat